bin/wifi

```diff
@@ -108,6 +109,9 @@ def autoconnect_command(args):
     else:
         assert False, "Couldn't find any schemes that are currently available."
 
+def disconnect_command(args):
+    subprocess.call(['dhclient', '-r', args.interface], stderr=subprocess.STDOUT)
```

Comment:
It would be nice not to repeat this code twice, but as there is no way to fetch the currently activated scheme, I'm ok with it.

What do you think?